docs/src/css/custom.css

```diff
@@ -29,6 +29,46 @@
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
 
+.navbar-github-link {
+  padding: 0px;
+  margin-right: 10px;
+}
+
+.navbar-docs-link {
+  padding: 0px;
+  margin-right: 10px;
+}
+
+.navbar-github-link::before {
```

Comment:
did you change the color of those icons in the navbar? I don't see the difference.

Reply:
ok never mind I see your comment. I still think it should be less contrasting, but I don't have a strong opinion, so it's up to you.